### ISSUE TITLE
[Feature request] Add 2FA parameters in backend user management

### ISSUE THREAD

**Issue by gibiwatch** [ssv445/Joomla-Two-Factor-Authentication#48]
It would be useful for site administrators to be able to see / edit 2FA parameters for any user in the backend user edit page, like happens for user profile plugins. Thanks.

**Comment by mManishTrivedi** [comment]
I think, you are right. Back-up code should be changed by admin /from backend. Actually when you enable 2FA system only for front-end user then you can't see it back-end user-manager screen.
As per your requirement, I need to always display 2FA tab in back-end. (into user-manager screen)
Can you please provide any feedback or input so we can improve UX. ??  :blush:

**Comment by vdrover** [comment]
I think you can keep it pretty simple. Here's the core Google Authenticator screen in the user manager:

http://take.ms/kw2L1

**Comment by mManishTrivedi** [comment]
@vdrover  : You are right but we are also covering J 2.5  where bootstrap is not available. :bowtie:  
We can separate out layout for Joomla 3.X and Joomla 2.5. 
Thanks for your feedback.
